src/DBFTPlugin/Messages/ConsensusMessage.cs

```diff
@@ -37,7 +37,7 @@ public static ConsensusMessage DeserializeFrom(byte[] data)
         {
             ConsensusMessageType type = (ConsensusMessageType)data[0];
             Type t = typeof(ConsensusMessage);
-            t = t.Assembly.GetType($"{t.Namespace}.{type}", false);
+            t = t.Assembly.GetType($"{t.Namespace}.{type.ToMessage()}", false);
             if (t is null) throw new FormatException();
```

Comment:
It should throw this exception if it was not found, could detail the error?

Reply:
This pr https://github.com/neo-project/neo-modules/pull/683

I added two new message types to do the DKG. The message name is DKGShareMessage while the type is DKGShare, the message could be sent, but the node can only receive one response message.


![image](https://user-images.githubusercontent.com/10189511/150705200-fa1e4889-4b9b-48c3-b391-f0f40a8c75df.png)

Reply:
Please, try with this https://github.com/neo-project/neo-modules/pull/685

Reply:
Using enumerated type name to represent the class name already break the basic rules of software pattern. What I learned from the painful debug experience in the last few days is that the current implementation is misleading.